utils/src/main/kotlin/LicenseFilenamePatterns.kt

```diff
@@ -44,4 +46,66 @@ object LicenseFilenamePatterns {
     val ROOT_LICENSE_FILENAMES = listOf(
         "readme*"
     ).generateCapitalizationVariants()
+
+    /**
+     * Return glob patterns which match all files which may contain license information residing recursively within the
+     * given absolute [directory] or in any of its ancestor directories.
+     */
+    fun getLicenseFileGlobsForDirectory(directory: String): List<String> =
+        globFileInDirectoryOrAncstors(directory, LICENSE_FILENAMES + ROOT_LICENSE_FILENAMES)
```

Comment:
So this returns license files up to the root directory of the filesystem? Shouldn't it stop at the root of the repository?

Reply:
Well, I'm only going to pass VCS paths as parameter.

Reply:
Then I got confused because the docs say "absolute directory".

Reply:
The directory is supposed to start with a '/'.